db/seeds.rb

```diff
@@ -34,6 +36,27 @@ def create_stickers
                                       image_url: "http://devstickers.com/assets/img/cat/chrome.png",
                                       price: 2,
                                       description: "Chrome logo")
+    @javascript_category.stickers.create(title: "Angularjs",
+                                         image_url: "http://devstickers.com/assets/img/cat/angularjs.png",
+                                         price: 3,
+                                         description: "Angular.js logo")
+    @javascript_category.stickers.create(title: "Javascript",
+                                         image_url: "http://devstickers.com/assets/img/cat/javascript.png",
+                                         price: 5,
+                                         description: "Javascript logo")
+    @git_category.stickers.create(title: "Git",
+                                   image_url: "http://devstickers.com/assets/img/cat/git.png",
```

Comment:
Line is too long. [94/80]<br>Align the elements of a hash literal if they span more than one line.